electron/libelectron/src/backends/photon.hh

```diff
@@ -17,16 +17,17 @@ private:
 public:
     PhotonBackend();
 
+    memory_mgmt::FirstFitAllocator *dev_mem_obj;
```

Comment:
```cpp
std::shared_ptr<memory_mgmt::FirstFitAllocator> allocator;
```

Reply:
(I think "allocator" is a more descriptive name here since it only allocates addresses, and isn't itself the owner of the device memory blob)

Reply:
Done